data/lua/wml-tags.lua

```diff
@@ -911,11 +911,10 @@ function wml_actions.role(cfg)
 
 	filter.role, filter.type = nil, nil
 	local search_map, search_recall, reassign = true, true, true
-	if cfg.search_recall_list ~= nil then
```

Comment:
Quick test only.

As to 'not not' read http://stackoverflow.com/questions/11806305/why-use-not-not-operation

Which leads me to think there's more I should test ...